### PR TITLE
Update workflow with DRAFT status

### PR DESCRIPTION
**Pull request by skoczko**

### CODE REVIEW

File: NEW_MOD_TECH_EVAL_JIRA.md [hunk 1]
```diff
@@ -10,16 +10,18 @@ This document outlines the JIRA Project and Workflow used by the New Module Tech
...
-**SUBMITTED** Initial status for newly created submissions.
+**DRAFT** Initial status for newly created submissions.
+
+**SUBMITTED** Once the submission is ready for the TC review, the submitter transitions the issue to this state.
 
 **UNDER EVALUATION** Once the sumbmission is reviewed by the TC and an evaluation group is formed, the TC indicates this by transitioning the issue to this state.
 
 **CANCELLED** If the submitter decides to withdraw the submission (alone or together with the evalation group), he/she transitions the issue to this state.
-The submisison may be re-opened at a later time.
+The submisison may be resumed at a later time.
```
[the remark this answers is not in the document]
Comment (zburke) [reply NEW_MOD_TECH_EVAL_JIRA.md:20]: I actually like the parity of "resubmitted" with the state "submitted". "Resumed" to me implies the process was paused in the middle, not that it entered a "final" state (such as "Cancelled" or "Rejected").

File: NEW_MOD_TECH_EVAL_JIRA.md [hunk 2]
```diff
@@ -10,16 +10,18 @@ This document outlines the JIRA Project and Workflow used by the New Module Tech
...
 
 **UNDER TC REVIEW** Once the evaluation is complete, the evalution group communicates the results and transitions the issue to this state.
 
 **APPROVED** Once the evalution results are reviewed by the TC and the submission is approved, the TC transtions the issue to this state.
 
 **REJECTED** Once the evalution results are reviewed by the TC and the submission is rejected, the TC transtions the issue to this state. 
-The submitter may re-submit the submission once the failed criteria are addressed.
+The submitter may resume the submission once the failed criteria are addressed.
```
Comment (zburke) [inline NEW_MOD_TECH_EVAL_JIRA.md:27]: As above, I actually like "resubmit".

**Review by zburke — CHANGES_REQUESTED** [review]
The only real request here is to correctly spell "submission" :) 

I personally like the phrase "resubmit" better than "resume", but I don't feel strongly about it.

File: NEW_MOD_TECH_EVAL_JIRA.md [hunk 3]
```diff
@@ -10,16 +10,18 @@ This document outlines the JIRA Project and Workflow used by the New Module Tech
 ## JIRA Workflow
 ![Techical Council Requests Workflow](assets/workflow.png)
 
-**SUBMITTED** Initial status for newly created submissions.
+**DRAFT** Initial status for newly created submissions.
+
+**SUBMITTED** Once the submission is ready for the TC review, the submitter transitions the issue to this state.
 
 **UNDER EVALUATION** Once the sumbmission is reviewed by the TC and an evaluation group is formed, the TC indicates this by transitioning the issue to this state.
```
Comment (craigmcnally) [inline NEW_MOD_TECH_EVAL_JIRA.md:17]:
```suggestion
**UNDER EVALUATION** Once the submission is reviewed by the TC and an evaluation group is formed, the TC indicates this by transitioning the issue to this state.
```

**Review by jeremythuff — APPROVED** [review]

**Review by craigmcnally — APPROVED** [review]

**Pull request merged**